### PR TITLE
Improve mainpage hero

### PR DESCRIPTION
**Pull request by MHerholdt94**
## Title

#774 

## Describe your changes: 

Improves the mainpage hero to be more eye-catching.

Changes include:
- Adds primary theme color to "Noroff Job Agency" in title
- Modifies text wrapping for title
- Splits title and subtitle (and fixes subtitle typo)
- Adds some text shadow
- Adds permanent "shadowed" background that originally only appeared on smaller screen sizes
- Centeres text and button
- Adds semibold text to button + adjusts width

## Type of changes:

Changes to Bootstrap classes and some SCSS styles as well as slight changes to HTML

## Checklist before requesting a review

* [x] I created a new branch before starting with this ticket?
* [x] I commented the issue after finish in the Frontend Team Board?
* [x] I moved the card to Quality Assurance after finish?

## Did you run into any issues?

- No issues encountered.

## Screenshots:

- Yes, the screenshots are included in the table below.

| Before | After |
:---------|--------:
mainpage_hero_before | mainpage_hero_after 
mainpage_hero_before_sm | mainpage_hero_after_sm

## Feedback

- Yes, I want feedback.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,127 +1,111 @@
...
+  <!-- Description -->
+  <meta name="description" content="" />
+  <title></title>
+  <!-- Links and scripts -->
 
-  <body id="homePage">
-    <header></header>
-    <main class="bg-theme-grey">
-      <!-- SECTION 1 -->
+  <script defer src="./index.js" type="module"></script>
```
Comment (Sovak3441) [inline index.html:13]: defer have no effect on module scripts. Meaning script tag with type="module" does not need to use defer as they defer by default.

**Review by Sovak3441 — CHANGES_REQUESTED** [review]
Please remove the refer for script module. See in-document comment

File: index.html (hunk 1 above)
Comment (MHerholdt94) [reply index.html]: done!

**Review by Sovak3441 — APPROVED** [review]
LGTM, Thanks for changing

**Pull request merged**